web_src/js/features/common-global.js

```diff
@@ -60,6 +60,7 @@ export function initGlobalEnterQuickSubmit() {
 export function initGlobalButtonClickOnEnter() {
   $(document).on('keypress', '.ui.button', (e) => {
     if (e.keyCode === 13 || e.keyCode === 32) { // enter key or space bar
+      if (e.target.nodeName === 'BUTTON') return; // button already handles space&enter correctly
```

Comment:
It may need more fine-tunes in the future , for example, IIRC `<a>` handles `Enter` but does not handle `Space`.

I can not remember correctly at the moment and haven't got enough time to do a full test, so I think maybe we can defer the changes to the future.